### PR TITLE
Updated refactor

### PR DESCRIPTION
**Pull request by Lach98**
updated refactoring branch resolves whatever conflict original refactor was having

### CODE REVIEW

File: navigation.php [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
-					<li><a href=\"displayItems.php\">DISPLAY ITEMS</a></li>
-		      <li><a href=\"monthly_report.php\">REPORTS</a></li>
-		    </ul>
+		    <div class='dropdown'>
+			    <button class='dropbtn'>SALES 
+			      <i class='fa fa-caret-down'></i>
+			    </button>
+			    <div class='dropdown-content'>
+			    <!-- ADD SALES PAGE LINKS HERE!!! -->
+			      <a href='add_sales.php'>ADD SALES</a>
```
Comment (dannamachi) [inline navigation.php]: link is addSalesOrStock.php here

**Review by dannamachi — CHANGES_REQUESTED** [review]
Works fine, dropdown looks nice but for some reason when you hover over Sales and Items, the other menu tabs move away from their original position.

Monthly report is bcs css is not linked so i will do that in my branch.

Also for sales it would be nice to change the menu name from Add Sales to Add Sales/Stock. One of the link in the dropdown has to be changed per the comment.

File: navigation.php (hunk 1 above)
Comment (Lach98) [reply navigation.php]: ah okay, I think mine was working from an older version, I copied across exact links. I'll give it a fix, thanks

**Review by dannamachi — APPROVED** [review]
Gotta fix floating later but otherwise good to go

**Pull request merged**